tutorials/classify-maizeland-ng/validation-pts.csv

```diff
@@ -0,0 +1,576 @@
+CRP,Lat,Lon,Other_Crop,RSP,Yield_t_ha,class,id,otherc

```

Comment:
Can we remove these files now that they're shared via Assets?

Reply:
Done